CHANGELOG.md

```diff
@@ -19,6 +20,9 @@ Use the following format for additions: ` - VERSION: [feature/patch (if applicab
 - Limit commit title to 72 characters, the rest is truncated and shown when inspecting the commit
 - Updated file watch logic to closely follow git commands in another process [#283](https://github.com/FredrikNoren/ungit/issues/283)
 
+### Changed
+- Refactored to use promises on client side [#852](https://github.com/FredrikNoren/ungit/pull/852)
```

Comment:
I was thinking we just do the one line changes (that you added on top), more details can be left in the complementing issue/PR